### DISCUSSION TITLE
Add automatically to list if created

### DISCUSSION THREAD

**Discussion by Monospace-V** [community/community#116617] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

If we do add to list and click the option to create a new list, then the repo should automatically add to the list we just created. I didn't realise it was separate till I opened the list dialogue box again.
Am using GitHub mobile.

**Comment by heart-king** [comment]
Hi ,help

**Reply by Mazzika1** [reply]
Hi

**Reply by Ihorog** [reply]
> Hi ,help
>

**Comment by Al3676** [comment]
Excellent

**Comment by huRfoj-mysfa8-rugqym** [comment]
[]()****

**Comment by EKartel** [comment]
🤫

**Reply by RemadanAdusebro** [reply]
Hi How are you?

**Comment by Randil-Hasanga** [comment]
- Open the GitHub app on your mobile device.
- Navigate to the repository you want to add to a list.
- Tap the three dots (ellipsis) in the upper-right corner of the repository page.
- Select “Add to list.”
- If you already have a list, choose it from the existing lists. Otherwise, select “Create a new list.”
- If you create a new list, give it a name and description.
- Confirm your selection, and the repository will be added to the list.

I hope this works

**Comment by subhadeeproy3902** [comment]
To add a repository to a list using the GitHub mobile app, follow these steps:

1. **Open the GitHub app** on your mobile device.
2. **Navigate to the repository** you want to add to a list.
3. **Tap the three dots (ellipsis)** in the upper-right corner of the repository page.
4. Select **“Add to list.”**
5. If you already have a list, **choose it from the existing lists**. Otherwise, select **“Create a new list.”**
6. If you create a new list, **give it a name and description**.
7. **Confirm your selection**, and the repository will be added to the list.

I hope this works!

**Comment by Famous-guy** [comment]
response #10134

**Comment by Somaticbeing84** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> If we do add to list and click the option to create a new list, then the repo should automatically add to the list we just created. I didn't realise it was separate till I opened the list dialogue box again.
> Am using GitHub mobile.
